lib/src/platform.dart

```diff
@@ -6,53 +6,38 @@
 
 import 'dart:io' show Platform;
```

Comment:
I haven't checked, but it seems that this import is not needed anymore. If it is, can you try to get rid of it? Flutter master will not build web projects with dart:io imports anymore.

Reply:
fixed and verified that it works in browser.